cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -147,7 +147,127 @@ property replaces the ``schedule`` property of an ETL Batch Application.
 
 In the example code above, we will use a *ProjectionTransform* (a type of Transform) to drop certain
 columns in the incoming data. A *StreamSink* in the final step needs a data field property
-that it will use as the content for the data to be written. 
+that it will use as the content for the data to be written.
+
+DAG in transform
```

Comment:
I don't know what this title is referring to. Can we come up with an alternative? It should be in "Capital Case".